Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -570,33 +570,57 @@ + (MAAdapterError *)toMaxError:(nullable NSError *)vungleError
             break;
         case 2: // Invalid AppID
         case 201: // Invalid PlacementID
+        case 207: // Invalid Placement Type
+        case 222: // Invalid Placement load type
         case 500: // BannerView: Invalid Size
             adapterError = MAAdapterError.invalidConfiguration;
             break;
+        case 119: // Json Encode Error
+            adapterError = MAAdapterError.internalError;
+            break;
+        case 202: // Ad already Consumed
+        case 203: // Ad is already loading
+        case 204: // Ad already loaded
+        case 205: // Ad is playing
+        case 206: // Ad already failed loading
+            adapterError = MAAdapterError.invalidLoadState;
+            break;
         case 210: // Ad Not Loaded
-            adapterError = MAAdapterError.noFill;
+            adapterError = isPlayFlow ? MAAdapterError.adNotReady : MAAdapterError.invalidLoadState;
```

Comment:
Why can't this just be adNotReady? If it is not the play flow and it is not loaded, then it is still not ready right?

Reply:
adNotReady MAX error should only be called in play ad flow.